From: Dimitri John Ledkov <xnox@ubuntu.com>
Date: Wed, 14 Feb 2018 20:34:55 +0000
Subject: test/test-functions: Debian/Ubuntu, now ship 95-dm-notify.rules,
 copy it

This fixes cryptsetup tests on recent Ubuntu/Debian systems (current
development series).

Bug-Launchpad: https://launchpad.net/bugs/1749432
(cherry picked from commit 7e026ca51e48b5beb50fc7745b8678a101413d78)
---
 test/test-functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test-functions b/test/test-functions
index 018bdca..ac3de81 100644
--- a/test/test-functions
+++ b/test/test-functions
@@ -314,7 +314,7 @@ install_dmevent() {
     if [[ "$LOOKS_LIKE_DEBIAN" ]]; then
         # dmsetup installs 55-dm and 60-persistent-storage-dm on Debian/Ubuntu
         # see https://anonscm.debian.org/cgit/pkg-lvm/lvm2.git/tree/debian/patches/0007-udev.patch
-        inst_rules 55-dm.rules 60-persistent-storage-dm.rules
+        inst_rules 55-dm.rules 60-persistent-storage-dm.rules 95-dm-notify.rules
     else
         inst_rules 10-dm.rules 13-dm-disk.rules 95-dm-notify.rules
     fi
